### PR TITLE
Release 0.1.4 Prep

### PR DESCRIPTION
**Pull request by jessicafalk**
#### Description
- made changes necessary to be compatible with latest master of the gdk-for-unity repo
- added support for connecting mobile devices to cloud deployments
- made the pinned version the latest master at the time of writing
- upgraded to 2018.3.2f1

#### Tests
Ran locally. Need to do full testing before release

#### Documentation
How is this documented (for example: release note, upgrade guide, feature page, in-code documentation)?

**Did you remember a changelog entry?**

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: workers/unity/Assets/Scenes/AndroidClientScene.unity [hunk 1]
```diff
@@ -276,8 +276,9 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   MaxConnectionAttempts: 3
-  UseExternalIp: 1
-  ipAddress: 
+  DevelopmentAuthToken: c0c96e31-ab96-47fe-ba38-ae8ff37009a4
+  ipAddress: 172.16.120.72
```
Comment (jamiebrynes7) [inline workers/unity/Assets/Scenes/AndroidClientScene.unity]: Intentional changes?

File: workers/unity/Assets/Scripts/Workers/AndroidClientWorkerConnector.cs [hunk 2]
```diff
@@ -15,12 +15,18 @@ public class AndroidClientWorkerConnector : MobileWorkerConnector
         public const string WorkerType = "AndroidClient";
 
         [SerializeField] private string ipAddress;
+        [SerializeField] private bool connectLocal;
```
Comment (jamiebrynes7) [inline workers/unity/Assets/Scripts/Workers/AndroidClientWorkerConnector.cs]: Nit: `shouldConnectLocally`

File: workers/unity/Assets/Scripts/Workers/UnityGameLogicConnector.cs [hunk 3]
```diff
@@ -29,16 +26,16 @@ private static EntityTemplate CreatePlayerEntityTemplate(string workerId, Improb
...
-                .SetEntityAclComponentWriteAccess(serverAttribute)
-                .AddPlayerLifecycleComponents(workerId, clientAttribute, serverAttribute)
-                .AddTransformSynchronizationComponents(clientAttribute);
+            var template = new EntityTemplate();
+            template.AddComponent(new Position.Snapshot(), clientAttribute);
+            template.AddComponent(new Metadata.Snapshot { EntityType = "Player" }, serverAttribute);
+            TransformSynchronizationHelper.AddTransformSynchronizationComponents(template, clientAttribute);
+            PlayerLifecycleHelper.AddPlayerLifecycleComponents(template, workerId, clientAttribute, serverAttribute);
+
+            template.SetReadAccess(clientAttribute, serverAttribute);
```
Comment (jamiebrynes7) [inline workers/unity/Assets/Scripts/Workers/UnityGameLogicConnector.cs]: Mobile clients?

Comment (jamiebrynes7) [reply workers/unity/Assets/Scripts/Workers/UnityGameLogicConnector.cs]: Also `clientAttribute` is wrong here as it refers to the specific client as opposed to all clients

File: workers/unity/Assets/Scripts/Workers/iOSClientWorkerConnector.cs [hunk 4]
```diff
@@ -14,12 +14,18 @@ public class iOSClientWorkerConnector : MobileWorkerConnector
         public const string WorkerType = "iOSClient";
 
         [SerializeField] private string ipAddress;
+        [SerializeField] private bool connectLocal;
```
Comment (jamiebrynes7) [inline workers/unity/Assets/Scripts/Workers/iOSClientWorkerConnector.cs]: `shouldConnectLocally` here too

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,7 @@
 ### Added
 
 - Added a metric sending system for load, FPS and Unity heap usage.
+- Added support for connecting mobile clients to cloud deployments.
```
Comment (jamiebrynes7) [inline CHANGELOG.md]: I don't think this belongs here 😉

Comment (jessicafalk) [reply CHANGELOG.md]: but I did add it in this PR ;)

Comment (jessicafalk) [reply CHANGELOG.md]: nvm, wrong line 🤦‍♂️  will fix that

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,15 @@
...
 
+### Added
+
+- Added support for connecting mobile clients to cloud deployments.
+
+### Changed
+
+- Upgraded the project to be compatible with `2018.3.2f1`.
+
+
```
Comment (paulbalaji) [inline CHANGELOG.md]: nit: extra line

**Pull request merged**